### PR TITLE
[DEVOPS-398] Script to launch demo cluster with new testnet genesis

### PR DESCRIPTION
**Pull request by rvl**
## Description

Adds a script which launches a demo cluster. The difference with the other demo script is that it uses the same method as used to launch the public testnet cluster.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-398

## Type of change
- [~] 🐞 Bug fix (non-breaking change which fixes an issue)
- [x] 🛠 New feature (non-breaking change which adds functionality)
- [~] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [x] 🔨 New or improved tests for existing code
- [~] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist
- [x] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [x] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [x] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## QA Steps

Run the demo script and see if it works.

### CODE REVIEW

**Comment by akegalj** [comment]
hm - whats up with the so big git diff

**Comment by rvl** [comment]
@akegalj The base branch was behind develop. I changed it to develop since devops-398-testnet got merged.

File: scripts/launch/demo-cluster/default.nix [hunk 1]
```diff
@@ -86,70 +102,82 @@ in pkgs.writeScript "demo-cluster" ''
...
+    cardano-node-simple $node_args &> ${stateDir}/logs/relay$i.txt &
     relay_pid[$i]=$!
 
   done
   ${ifWallet ''
-    export LC_ALL=C.UTF-8
+    export LC_ALL=C
     echo Launching wallet node: ${demoWallet}
-    ${demoWallet} --runtime-args "--system-start $system_start" &> /dev/null &
+    ${demoWallet} --runtime-args "--system-start $system_start" &> ${stateDir}/logs/wallet.txt &
```
Comment (disassembler) [inline scripts/launch/demo-cluster/default.nix]: why txt instead of log extension?

Comment (disassembler) [reply scripts/launch/demo-cluster/default.nix]: There's a few other places where you have .txt extension to, that applies to all of them.

**Pull request merged**